docs/content/docs/connectors/table/formats/debezium.md

```diff
@@ -44,7 +44,7 @@ However, currently Flink can't combine UPDATE_BEFORE and UPDATE_AFTER into a sin
 Dependencies
 ------------
 
-#### Debezium Avro
+#### Debezium Avro Confluent
```

Comment:
should it be "Confluent Avro" instead of "Avro Confluent" for the consistency because of https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/connectors/table/formats/avro-confluent/

Reply:
Sure thing, absolutely correct